src/www/app/persistent_server.ts

```diff
@@ -33,8 +33,8 @@ export type PersistentServerFactory =
 export class PersistentServerRepository implements ServerRepository {
   // Name by which servers are saved to storage.
   private static readonly SERVERS_STORAGE_KEY = 'servers';
-  private serverById: Map<string, PersistentServer>;
-  private lastForgottenServer: PersistentServer|null;
+  private serverById = new Map<string, PersistentServer>();
```

Comment:
I think it's better to keep initializing in `loadServers()` as a defensive strategy, since calling it twice would be fine.